pkg/eval/builtin_special.go

```diff
@@ -36,9 +36,9 @@ var builtinSpecials map[string]compileBuiltin
 // intended for external consumption, e.g. the syntax highlighter.
 var IsBuiltinSpecial = map[string]bool{}
 
-type noSuchModule struct{ spec string }
+type NoSuchModule struct{ spec string }
```

Comment:
This needs a godoc now that it is exported.

Reply:
done